PlatformTools/root_dbuild.sh

```diff
@@ -1,6 +1,6 @@
-# ********  DO NOT EDIT THIS FILE  ********
-#/******************************************************************************
-#  Copyright (C) 2021-2024 Advanced Micro Devices, Inc. All rights reserved.
-#*****************************************************************************/
-# This file is in Platform\PlatformTools\root_dbuild.sh
-python3 Platform/PlatformTools/Tools/dbuild/dbuild.py "$@"
+# ********  DO NOT EDIT THIS FILE  ********
+#/******************************************************************************
+#  Copyright (C) 2021-2024 Advanced Micro Devices, Inc. All rights reserved.
+#*****************************************************************************/
+# This file is in Platform\PlatformTools\root_dbuild.sh
+python3 Platform/PlatformTools/Tools/dbuild/dbuild.py "$@"
```

Comment:
Add a # at the end of the line so that the \r doesn't trip things up:
`python3 Platform/PlatformTools/Tools/dbuild/dbuild.py "$@" #`

Otherwise you get this:
```
./dbuild.sh genoa-onyx
usage: dbuild.py [-h] [-v {CRITICAL,ERROR,WARNING,INFO,DEBUG}] {genoa-onyx,unit,clean} ...
dbuild.py: error: argument command: invalid choice: 'genoa-onyx\r' (choose from 'genoa-onyx', 'unit', 'clean')
```

Reply:
That is already done in this PR
The point is that the `genoa_poc.yml` is referencing the original repo.
I have tested by changing the url to point to this repo and works good.